src/components/skills-quiz/SearchJobDropdown.jsx

```diff
@@ -1,7 +1,41 @@
-import React from 'react';
+import React, { useContext, useMemo } from 'react';
+import { SearchContext } from '@edx/frontend-enterprise-catalog-search';
+import FacetListRefinement from '@edx/frontend-enterprise-catalog-search/FacetListRefinement';
 
-const SearchJobDropdown = () => (
-  <p>Search job dropdown here!</p>
-);
+import { JOBS_QUIZ_FACET_FILTERS } from './constants';
+
+const SearchJobDropdown = () => {
+  const { refinementsFromQueryParams } = useContext(SearchContext);
+  const { title, attribute, typeaheadOptions } = JOBS_QUIZ_FACET_FILTERS;
+  const skillQuizFacets = useMemo(
```

Comment:
isn't this `jobsQuizFacet` ?

Reply:
I think a better name would be `jobsDropdown`

Reply:
Good catch